docs/resources/server/enchantments.md

```diff
@@ -0,0 +1,85 @@
+# Enchantments
+
+Enchantments are enchancements that can be applied to tools and other items. As of 1.21, enchantments are stored on items as [Data Components], are defined in JSON, and are comprised of Enchantment Components.
```

Comment:
Enchantment effect components.

Reply:
Right, that is their name isn't it... I've made this adjustment everywhere.